data/json/npcs/lumbermill employees/lumbermill missions.json

```diff
@@ -0,0 +1,140 @@
+	  "advice": "Dismantling a building somewhere in town could get you wnat you need.",
+      "accepted": "I'll be here when you get back. Good luck!",
+      "rejected": "Well, I suppose that's reasonable.",
+      "inquire": "Do you have the nails?",
+      "success": "Thats absolutely amazing! I can't thank you enough for this!",
+      "success_lie": "You fucking scammer <name_b>!",
+      "failure": "That's unfortunate. I guess we'll have to find someone else to do it."
+    }
+  },
+{
+    "id": "MISSION_LUMBERMILL_GET_6_BLANKETS",
+    "type": "mission_definition",
+    "name": "Find 6 Blankets",
+    "description": "Obtain 6 blankets in exchange for credit with the loggers.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "blanket",
+	"count": 6,
+    "difficulty": 5,
+    "value": 4000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],          
+    "end": {
+	    "update_mapgen": { "om_terrain": "lumbermill_1_1_ocu", "place_nested": [ { "chunks": [ "occupied_lum_hotel_1_1" ], "x": 0, "y": 0 } ] }
+           },
+    "followup": "MISSION_LUMBERMILL_SET_TRADE_ROUTE",   
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "I need 6 blankets.",
+	  "offer": "Now that we should be safe from harm, I decided to start adding a few things to this place. Right now, we don't have anywhere to comfortably sleep, so we're building a small sleeping quarter. If you could get us 6 blankets, we could finish our project.",
+	  "advice": "A hotel in town should have lots of them.",
+      "accepted": "I'll be here when you get back. Good luck!",
+      "rejected": "Someone's got to do it.",
+      "inquire": "Where are the blankets?",
+      "success": "Thanks! Feel free to sleep there if you want.",
+      "success_lie": "How does this help us?",
+      "failure": "That's unfortunate. I guess we'll have to find someone else to do it."
+    }
+  },
+{
+    "id": "MISSION_LUMBERMILL_SET_TRADE_ROUTE",
+    "type": "mission_definition",
+    "name": "Establish A Trade Route",
+    "description": "Establish trade relations between the loggers and a refugee center.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "trade_writ",
+	"count": 1,
+    "difficulty": 35,
+    "value": 4000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],  
+        "start": {
+                  "assign_mission_target": { "om_terrain": "evac_center_18", "reveal_radius": 0 }
+                 },
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "Go negotiate a trade agreement.",
+	  "offer": "While some of us were out scavenging, we came across an old adress for an evac center; and from what we found out, there should be some people there. I want you to go over there, see whoever runs the joint and negotiate a deal with them for refined lumber. This will be invaluble to our survival.",

```

Comment:
```suggestion
	  "offer": "While some of us were out scavenging, we came across an old address for an evac center; and from what we found out, there should be some people there. I want you to go over there, see whoever runs the joint and negotiate a deal with them for refined lumber. This will be invaluable to our survival.",

```